### PR TITLE
Change color of cancel staff infraction button from green to gray

### LINKED ISSUE #2978: Update ban confirmation button labels and colours
With PR #2316  merged, the bot will now ask for confirmation when banning members with elevated roles. 

Based on feedback from staff, it has been suggested to change the button labels to "Ban" (in red) and "Cancel" (in gray) since the current colour scheme might be a bit confusing.

### PR DESCRIPTION
**Pull request by Robin5605**
Closes #2978

### CODE REVIEW

**Review by vivekashok1221 — APPROVED** [review]
Thanks :star2:

File: bot/exts/moderation/infraction/_views.py [hunk 1]
```diff
@@ -13,14 +13,14 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
         self.confirmed = False
 
-    @discord.ui.button(label="Confirm", style=ButtonStyle.red)
+    @discord.ui.button(label="Ban", style=ButtonStyle.red)
     async def confirm(self, interaction: Interaction, button: Button) -> None:
         """Callback coroutine that is called when the "confirm" button is pressed."""
```
Comment (vivekashok1221) [inline bot/exts/moderation/infraction/_views.py:18]:
```suggestion
        """Callback coroutine that is called when the "Ban" button is pressed."""
```
or 
```suggestion
        """Callback coroutine that is called when the button labelled "Ban" is pressed."""
```

**Review by vivekashok1221 — CHANGES_REQUESTED** [review]
Slight nit.

**Review by ichard26 — COMMENTED** [review]
The view class was intentionally designed to be generic, able to be used to request confirmation for any infraction. Consequently, it was named `InfractionConfirmationView`. This change, while a change for the better IMO, restricts the class to ban infractions.

Can we please rename the class to `BanConfirmationView` at least?

**Comment by Robin5605** [comment]
@wookie184 I've addressed @ichard26's comment and @vivekashok1221's review. Should be good to go now, let me know if I've missed anything.

**Review by vivekashok1221 — APPROVED** [review]
LGTM :star2:

**Pull request merged**